components/brave_rewards/resources/page/actions/rewards_actions.ts

```diff
@@ -72,6 +56,8 @@ export const onModalBackupClose = () => action(types.ON_MODAL_BACKUP_CLOSE)
 
 export const onModalBackupOpen = () => action(types.ON_MODAL_BACKUP_OPEN)
 
+export const setBackupCompleted = () => action(types.SET_BACKUP_COMPLETED)
```

Comment:
Hmm, this should be removed (along with `types.SET_BACKUP_COMPLETED`, and any related code).

Reply:
Fixed.